tasmota/tasmota_xdrv_driver/xdrv_79_esp32_ble.ino

```diff
@@ -2981,7 +2981,7 @@ void CmndBLEAlias(void){
           return;
         }
 
-        AddLog(LOG_LEVEL_ERROR,PSTR("BLE: Add Alias mac %s = name %s"), mac, p);
+        AddLog(LOG_LEVEL_INFO, PSTR("BLE: Add Alias mac %s = name %s"), mac, p);
```

Comment:
This change does not belong in the MR.

Reply:
Found while testing, so fixed it directly.

Reply:
What's the reason for changing the log level?

Reply:
Adding a BLE alias is not an error, just an info.